jmclient/jmclient/configure.py

```diff
@@ -612,10 +593,26 @@ def get_mchannels(mode="TAKER"):
             onion_data["serving"] = False
         onion_data['btcnet'] = get_network()
         # Just to allow a dynamic set of var:
-        onion_data["section-name"] = section
-        configs.append(onion_data)
+        onion_data["section-name"] = s
+        return onion_data
 
-    return configs
+    # processing the IRC sections:
```

Comment:
Comment can be removed

Reply:
Done in https://github.com/JoinMarket-Org/joinmarket-clientserver/pull/1182/commits/6a4cf97887a1abfd172362cced22636527bead41